packages/schematics/angular/workspace/schema.json

```diff
@@ -33,7 +33,7 @@
       "x-user-analytics": 14
     },
     "strict": {
-      "description": "Creates a workspace with stricter TypeScript compiler options.",
+      "description": "Creates a workspace with stricter type checking options.",
```

Comment:
```suggestion
      "description": "Creates a workspace with stricter type checking and build optimization options.",

```

Reply:
I prefer to leave this as original, as workspace doesn't have any build settings.

- `strict` in application enables striker build optimisations rules 
- `strict` in a workspace enables stricter type checking for all projects

Reply:
I'm fine with keeping the original.